content/chapters/compute/lab/support/sum-array/c/sum_array_processes.c

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <sys/time.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/mman.h>
+#include <sys/wait.h>
+
+#include "generate_random_array.h"
+
+#define ARR_LEN 100000000
+
+#define min(a, b)                       \
+	({                              \
+		__typeof__(a) _a = (a); \
+		__typeof__(b) _b = (b); \
+		_a < _b ? _a : _b;      \
+	})
```

Comment:
```suggestion
#define min(a, b)			\
	({				\
		__typeof__(a) _a = (a);	\
		__typeof__(b) _b = (b);	\
		_a < _b ? _a : _b;	\
	})
```
Tabs, not spaces. And move this and `ARRAY_LEN` to a local `array_utils.h` file, because they're used by every code snipped.